realm/realm-library/src/main/cpp/io_realm_internal_TableQuery.cpp

```diff
@@ -19,9 +19,11 @@
 #include <shared_realm.hpp>
 #include <object_store.hpp>
 #include "util.hpp"
+#include "jni_util/java_local_ref.hpp"
 #include "io_realm_internal_TableQuery.h"
 
 using namespace realm;
+using namespace realm::jni_util;
```

Comment:
Never `using` in header files - agree.
It's not a convention to use ùsing` in .cpp files. It's more a convenience for often used methods. In this case it seems better to fully qualify as it's rather rarely used in the files anyway.